src/content/zh-tw/fundamentals/primers/async-functions.md

```diff
@@ -0,0 +1,346 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description:異步函數用於像編寫同步代碼那樣編寫基於 Promise 的代碼
+
+{# wf_published_on: 2016-10-20 #}
+{# wf_updated_on: 2016-10-20 #}
+
+# 異步函數 - 提高 Promise 的易用性 {: .page-title }
+
+{% include "web/_shared/contributors/jakearchibald.html" %}
+
+Chrome 55 中默認情況下啓用異步函數，坦率地講，它們的作用相當不可思議。
+可以利用它們像編寫同步代碼那樣編寫基於 Promise 的代碼，而且還不會阻塞主線程。
+它們可以讓異步代碼“智商”下降、可讀性提高。
+
+
+異步函數的工作方式是這樣的：
+
+    async function myFirstAsyncFunction() {
+      try {
+        const fulfilledValue = await promise;
+      }
+      catch (rejectedValue) {
+        // …
+      }
+    }
+
+如果在函數定義之前使用了 `async` 關鍵字，就可以在函數內使用 `await`。
+當您 `await` 某個 Promise 時，函數暫停執行，直至該 Promise 產生結果，並且暫停並不會阻塞主線程。
+如果 Promise 執行，則會返回值。
+如果 Promise 拒絕，則會拋出拒絕的值。
+
+注：如果不熟悉 Promise，可以看一看[我們的 Promise 指南](/web/fundamentals/getting-started/primers/promises)。
```

Comment:
Don't translate [callouts](https://developers.google.com/web/resources/markdown-syntax#callouts) like `Note:` as it's a special marker for DevSite. The content should be translated, but the callout itself, shouldn't.

Reply:
Oops. (it's directly translated from zh-cn anyway, i will fix that later)